Adds waitress and waitress bonus

dan-snw opened the pull request:
Fixes #5 

## Changes

Adds a waitress and waitress bonus. Formatting not yet done hence a draft request.

dan-snw committed:
Adds waitress and waitress bonus

Formatting not done well yet but the buttons are
logic are there.

egnwd commented on package-lock.json:
You’ll want to delete this and run yarn

dan-snw replied:
Oh sorry I missed this when rebasing, it's now deleted.

egnwd commented on src/Components/HouseDemandContext.tsx:
```diff
@@ -27,6 +27,10 @@ export type HouseDemand = {
   setHasGarden: React.Dispatch<React.SetStateAction<boolean>>;
   hasCFO: boolean;
   setHasCFO: React.Dispatch<React.SetStateAction<boolean>>;
+  hasFirstWaitress : boolean;
```
First waitress is a milestone so should be included in that object

dan-snw replied:
Good point, it's resolved.

egnwd reviewed (changes requested):
Looks good, just a couple of small tweaks

dan-snw committed:
Removes package-lock.json

Accidentally kept after pulling from the upstream repo post change to
yarn - now deleted to avoid npm conflict.

dan-snw committed:
Makes first waitress a milestone

Rather than it being a separate boolean, it's just part of the
milestones object. The result is the same but it's more intuitive and
avoids the need for an entirely different object to be created.
